### PR TITLE
Fix sorts in code examples

### PR DESCRIPTION
**Pull request by bradenbiz**

### CODE REVIEW

**Review by blackbaud-johnly — APPROVED** [review]

File: projects/ag-grid/documentation/code-examples/data-grid/data-manager-added/data-manager-data-grid-docs-demo-view-grid.component.ts [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+  public sortItems(): void {
+    let sortOption = this.dataState.activeSortOption;
+    if (this.columnApi && sortOption) {
+      const sort = [
+        {
+          colId: sortOption.propertyName,
+          sort: sortOption.descending ? 'desc' : 'asc',
+        },
+      ];
+      this.gridApi?.setSortModel(sort);
```
Comment (johnhwhite) [inline projects/ag-grid/documentation/code-examples/data-grid/data-manager-added/data-manager-data-grid-docs-demo-view-grid.component.ts:235]: I'm getting errors
```
AG Grid: as of version 24.0.0, setSortModel() is deprecated, sort information is now part of Column State. Please use columnApi.applyColumnState() instead.
```

File: projects/ag-grid/documentation/code-examples/data-entry-grid/data-manager-added/data-manager-data-entry-grid-docs-demo-view-grid.component.ts [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+  public sortItems(): void {
+    let sortOption = this.dataState.activeSortOption;
+    if (this.columnApi && sortOption) {
+      const sort = [
+        {
+          colId: sortOption.propertyName,
+          sort: sortOption.descending ? 'desc' : 'asc',
+        },
+      ];
+      this.gridApi?.setSortModel(sort);
```
Comment (johnhwhite) [inline projects/ag-grid/documentation/code-examples/data-entry-grid/data-manager-added/data-manager-data-entry-grid-docs-demo-view-grid.component.ts:272]: Again, I'm seeing errors
```
AG Grid: as of version 24.0.0, setSortModel() is deprecated, sort information is now part of Column State. Please use columnApi.applyColumnState() instead.
```

**Review by johnhwhite — CHANGES_REQUESTED** [review]
I'm seeing deprecation warnings from AG Grid (see below) and an errror:
```
ERROR
Error: StaticInjectorError(AppModule)[DragulaDirective -> ElementRef]:
StaticInjectorError(Platform: core)[DragulaDirective -> ElementRef]:
NullInjectorError: No provider for ElementRef!
```

**Comment by bradenbiz** [comment]
> I'm seeing deprecation warnings from AG Grid (see below) and an errror:
> 
> ```
> ERROR
> Error: StaticInjectorError(AppModule)[DragulaDirective -> ElementRef]:
> StaticInjectorError(Platform: core)[DragulaDirective -> ElementRef]:
> NullInjectorError: No provider for ElementRef!
> ```

this error has been documented in its own right [here](https://blackbaud.visualstudio.com/Products/_workitems/edit/2030855)

File: projects/ag-grid/documentation/code-examples/data-grid/data-manager-added/data-manager-data-grid-docs-demo-view-grid.component.ts (hunk 1 above)
Comment (bradenbiz) [reply projects/ag-grid/documentation/code-examples/data-grid/data-manager-added/data-manager-data-grid-docs-demo-view-grid.component.ts]: fixed in commit [17030aa - _"fix sort deprecation warning"_](https://github.com/blackbaud/skyux-ag-grid/pull/194/commits/17030aaa1a2498cd47cc247bbed2d42bb5ad7f88)

File: projects/ag-grid/documentation/code-examples/data-entry-grid/data-manager-added/data-manager-data-entry-grid-docs-demo-view-grid.component.ts (hunk 2 above)
Comment (bradenbiz) [reply projects/ag-grid/documentation/code-examples/data-entry-grid/data-manager-added/data-manager-data-entry-grid-docs-demo-view-grid.component.ts]: fixed in commit [17030aa - _"fix sort deprecation warning"_](https://github.com/blackbaud/skyux-ag-grid/pull/194/commits/17030aaa1a2498cd47cc247bbed2d42bb5ad7f88)

**Review by johnhwhite — CHANGES_REQUESTED** [review]

**Review by johnhwhite — APPROVED** [review]

**Pull request merged**